clients/privacy-center/components/modals/privacy-request-modal/PrivacyRequestForm.tsx

```diff
@@ -80,13 +89,21 @@ const usePrivacyRequestForm = ({
 
       // extract identity input values
       const identityInputValues = Object.fromEntries(
-        Object.entries(action.identity_inputs ?? {}).map(([key, field]) => {
-          const value = fallbackNull(values[key]);
-          if (typeof field === "string") {
-            return [key, value];
-          }
-          return [key, { label: field.label, value }];
-        })
+        Object.entries(action.identity_inputs ?? {})
+          // we have to support name as an identity_input for legacy purposes
+          // but we ignore it since it's not unique enough to be treated as an identity
+          .filter(([key]) => key !== "name")
+          .map(([key, field]) => {
+            const value = fallbackNull(values[key]);
+            if (typeof field === "string") {
+              if (key === "phone") {
+                // eslint-disable-next-line no-param-reassign
+                key = "phone_number";
+              }
```

Comment:
The key is `phone` in `config.json` but it needs to be converted to `phone_number` to be posted

Reply:
Oof. That's messy. The `IDENTITY_INPUTS_MAP` suggestion I had above was to try to help make this _slightly_ less magical, but it's still a confusing DX